recipes/trame-client/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "trame-client" %}
+{% set version = "2.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/trame-client-{{ version }}.tar.gz
+  sha256: ae7f08b6927ea69b7ab3dab71813da3fb4da5955cb037933554f9825f91ff92b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - trame
+    - trame_client
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/trame-client/
+  summary: Internal client of trame
+  license: MIT
+  license_file:
+    - LICENSE
+    - trame_client/LICENSE
+
+extra:
+  recipe-maintainers:
+    - larsoner
+    - banesullivan
```

Comment:
@banesullivan do you agree to co-maintain the recipe?

Reply:
@jourdain and @psavery let me know if you'd like to be added

Reply:
Sure, that seems reasonable. Thanks!

Reply:
Yes, thanks @larsoner!